### PR TITLE
Drop partially-identified taxa (see #35)

### PR DESCRIPTION
**Pull request by davharris**

### CODE REVIEW

File: forecast-bbs-core.R [hunk 1]
```diff
@@ -28,18 +28,36 @@ install_dataset <- function(dataset){
...
+#' (e.g. "sp.") or were considered hybrids (e.g. "A x B").
 #'
 #' @param df dataframe containing an species_id column
 #'
 #' @return dataframe, filtered version of initial dataframe
 filter_species <- function(df){
-  bbs_data_filtered <- df %>%
+  
+  con <- dbConnect(RPostgres::Postgres(), dbname = 'postgres', 
+                   host = "localhost", password = "p", user = "postgres")
```
Comment (ethanwhite) [inline forecast-bbs-core.R]: I'd like to avoid explicitly passing credential here if possible. Are you having trouble getting it to work without Line 40?

**Comment by ethanwhite** [comment]
I just add #38, so if the easiest way around this credential issue is to go ahead and make the move here that would be fine.

File: forecast-bbs-core.R (hunk 1 above)
Comment (davharris) [inline forecast-bbs-core.R]: Oops, yeah.  Good catch. I finally got RPostgres to recognize my host (with PGHOST) and password (with . pgpass), but it still defaults to using "davidharris" as my username even though:

* I have `export PGUSER=postgres` in my .bash_profile
* When I run `psql` from the command line, it respects PGUSER

Anyway, I'll push a new commit without the credentials.

**Comment by ethanwhite** [comment]
Thanks @davharris.

**Pull request merged**